planner/pom.xml

```diff
@@ -44,7 +44,7 @@ limitations under the License.
 
     </dependencies>
 
-    <repositories>
+     <repositories>
```

Comment:
It was not my intention to change the pom.xml of planner. It looks like a white space was accidentally included.